gamemodes/terrortown/gamemode/server/sv_voice.lua

```diff
@@ -128,7 +143,7 @@ local function RoleGlobalVoice(ply, isGlobal)
 	local tm = ply:GetTeam()
 
 	ply[tm .. "_gvoice"] = isGlobal
-	ply.blockVoice = hook.Run("TTT2CanUseVoiceChat", ply, tm)
+	ply.blockVoice = hook.Run("TTT2CanUseVoiceChat", ply, not isGlobal) == false
```

Comment:
Where is this being used?

Reply:
fabian added this hook a few days ago and added `tm` on server, while having a bool on clients. He said this was a bug

Reply:
So blockvoice is used, right? I‘m just wanna keep sure that there aren‘t any inconsistencies or unnecessary vars

Reply:
I have no idea, but I think so. He said there was a double bug (missing `== false` and wrong variable) which lead to it still working

Reply:
@menof36go maybe this should clean up the situation :D

Reply:
blockVoice can be used to block a sender from using the voice chat. This could easily be used to replace the wrong hook the spy was using for example. blockVoice itself is used in PlayerCanHearPlayersVoice. It has to be a flag, otherwhise we would have to run the hook for every listener as well, which is quite costly